<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter</class>
 <widget class="QWidget" name="filter">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>597</width>
    <height>298</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelFilterItems">
         <property name="text">
          <string>Path Filter Items:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combItems">
         <property name="currentText">
          <string/>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAfterCurrent</enum>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFilterSettings">
         <property name="text">
          <string>Filter Settings:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="singleValue">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="labelConstraint">
             <property name="text">
              <string>Constraint:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leExp"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Scalar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="doubleValue">
          <layout class="QFormLayout" name="formLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="labelConstraintX">
             <property name="text">
              <string>Constraint x:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leExpX"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelConstraintY">
             <property name="text">
              <string>Constraint y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leExpY"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Point2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="cbPoint2">
             <property name="text">
              <string>x == y</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="dribbleValue">
          <layout class="QFormLayout" name="formLayout_4">
           <item row="1" column="0">
            <widget class="QLabel" name="labelConstraintX_2">
             <property name="text">
              <string>Constraint x:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leExpX_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelConstraintY_2">
             <property name="text">
              <string>Constraint y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leExpY_2"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelConstraintZ">
             <property name="text">
              <string>Constraint z:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="leExpZ"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Point3</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="cbPoint3">
             <property name="text">
              <string>x == y == z</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="colorValue">
          <layout class="QFormLayout" name="formLayout_5">
           <item row="1" column="0">
            <widget class="QLabel" name="labelConstraintRed">
             <property name="text">
              <string>Constraint Red:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leExpR"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelConstraintGreen">
             <property name="text">
              <string>Constraint Green:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leExpG"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelConstraintBlue">
             <property name="text">
              <string>Constraint Blue:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="leExpB"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="cbColor">
             <property name="text">
              <string>r == g == b</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="filterList"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cbFilterEnabled">
       <property name="text">
        <string>Filters enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddFilter">
       <property name="text">
        <string>Add Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApplyFilter">
       <property name="text">
        <string>Apply Filter(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteFilter">
       <property name="text">
        <string>Delete Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearAll">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
